ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -212,6 +215,33 @@ public enum CocoaPodUtils {
     }
   }
 
+  /// Get all transitive pod dependencies for a pod.
+  /// - Returns: An array of Strings
```

Comment:
What is the content of the strings, pod names?

Reply:
Yep. Clarified.